src/js/components/message.jsx

```diff
@@ -35,16 +36,54 @@ export class MessageComponent extends Component {
         }
 
         const style = {};
-        if (isAppUser && this.props.accentColor) {
+
+        if (!this.props.mediaUrl) {
+          style.borderTopLeftRadius = style.borderTopRightRadius = style.borderBottomLeftRadius = style.borderBottomRightRadius = 16;
```

Comment:
Can we extract the magic numbers `16` and `2` into constants at the top of the file?